`shouldContainAll` -> is the output of `expected` and `but was`  incorrectly swapped?

Kotest version `5.9.1`

Is the output of `expected` and `but was` in the `shouldContainAll` check incorrectly swapped?

```kotlin
priceSettings.priceModifiersWoodType shouldContainAll priceSettingsDto.priceModifiersWoodType.map {
        it.toModel()
      }
```

```
Collection should contain all of [PriceModifierWoodType(value=10.0099999999999997868371792719699442386627197265625, woodType=ASH), PriceModifierWoodType(value=20.00, woodType=OAK)] but was missing [PriceModifierWoodType(value=10.0099999999999997868371792719699442386627197265625, woodType=ASH)]Possible matches:
java.lang.AssertionError: Collection should contain all of [PriceModifierWoodType(value=10.0099999999999997868371792719699442386627197265625, woodType=ASH), PriceModifierWoodType(value=20.00, woodType=OAK)] but was missing [PriceModifierWoodType(value=10.0099999999999997868371792719699442386627197265625, woodType=ASH)]Possible matches:
 expected: PriceModifierWoodType(value=10.01, woodType=ASH),
  but was: PriceModifierWoodType(value=10.0099999999999997868371792719699442386627197265625, woodType=ASH),
```

the `value=10.0099999999..` is in `priceSettingsDto`. The `priceSettings` is the one, which is tested.

If it is a mistake I am willing to look for the bug and create a PR with a fix.

--- a/kotest-assertions/kotest-assertions-core/src/commonMain/kotlin/io/kotest/matchers/collections/containAll.kt
+++ b/kotest-assertions/kotest-assertions-core/src/commonMain/kotlin/io/kotest/matchers/collections/containAll.kt
@@ -39,7 +39,7 @@ fun <T> containAll(
          }
          val passed = missing.isEmpty()
 
-         val possibleMatchesDescription = possibleMatchesForSet(passed, value.toSet(), missing.toSet(), verifier)
+         val possibleMatchesDescription = possibleMatchesForSet(passed, missing.toSet(), value.toSet(), verifier)
 
          val failure =
             { "Collection should contain all of ${ts.print().value} but was missing ${missing.print().value}$possibleMatchesDescription" }

--- a/kotest-assertions/kotest-assertions-core/src/jvmTest/kotlin/com/sksamuel/kotest/matchers/collections/ShouldContainAllTest.kt
+++ b/kotest-assertions/kotest-assertions-core/src/jvmTest/kotlin/com/sksamuel/kotest/matchers/collections/ShouldContainAllTest.kt
@@ -127,10 +127,16 @@ class ShouldContainAllTest : WordSpec() {
                )
             }.shouldHaveMessage("""
                |Collection should contain all of [Fruit(name=apple, color=green, taste=sweet), Fruit(name=apple, color=red, taste=sweet)] but was missing [Fruit(name=apple, color=red, taste=sweet)]Possible matches:
-               | expected: Fruit(name=apple, color=green, taste=sweet),
-               |  but was: Fruit(name=apple, color=red, taste=sweet),
+               | expected: Fruit(name=apple, color=red, taste=sweet),
+               |  but was: Fruit(name=apple, color=green, taste=sweet),
                |  The following fields did not match:
-               |    "color" expected: <"green">, but was: <"red">
+               |    "color" expected: <"red">, but was: <"green">
+               |
+               | expected: Fruit(name=apple, color=red, taste=sweet),
+               |  but was: Fruit(name=pear, color=green, taste=sweet),
+               |  The following fields did not match:
+               |    "name" expected: <"apple">, but was: <"pear">
+               |        "color" expected: <"red">, but was: <"green">
     """.trimMargin())
          }
 
@@ -140,15 +146,15 @@ class ShouldContainAllTest : WordSpec() {
                   listOf(sweetGreenApple, sourYellowLemon)
                )
             }.message.shouldEndWith("""
-               | expected: Fruit(name=apple, color=red, taste=sweet),
-               |  but was: Fruit(name=apple, color=green, taste=sweet),
+               | expected: Fruit(name=apple, color=green, taste=sweet),
+               |  but was: Fruit(name=apple, color=red, taste=sweet),
                |  The following fields did not match:
-               |    "color" expected: <"red">, but was: <"green">
+               |    "color" expected: <"green">, but was: <"red">
                |
-               | expected: Fruit(name=pear, color=green, taste=sweet),
-               |  but was: Fruit(name=apple, color=green, taste=sweet),
+               | expected: Fruit(name=apple, color=green, taste=sweet),
+               |  but was: Fruit(name=pear, color=green, taste=sweet),
                |  The following fields did not match:
-               |    "name" expected: <"pear">, but was: <"apple">
+               |    "name" expected: <"apple">, but was: <"pear">
     """.trimMargin())
          }
       }

